### DISCUSSION TITLE
image stratch property is not working properly

### DISCUSSION THREAD

**Discussion by amit7775** [unoplatform/uno#8141] [Q&A]
image stratch property is not working properly
 <Image Grid.Row="0" Grid.Column="1" x:Name="imgbubble" Stretch="Uniform" Source="/Assets/Images/Union 2.svg"></Image>

**Comment by jeromelaban** [comment]
Thanks, can you be more specific? Best would be to post a sample project with the code which is not working.

**Reply by amit7775** [reply]
<Image Grid.Row="0" Grid.Column="1" Grid.RowSpan="6" x:Name="img1" Stretch="UniformToFill" Source="/Assets/Images/Image -1.svg" Margin="0,35,0,5"></Image>

Its not working on different scale and layout. it's working fine on 150% scale but not working fine on 125% or 100% scale.

**Reply by jeromelaban** [reply]
Thanks for the update. We'll needs screenshots and a repro sample to be able to help you. See the "how to reproduce it" [section here](https://github.com/unoplatform/uno/blob/master/.github/ISSUE_TEMPLATE/bug-report.yml).
